docs/HyperSync/api-tokens.mdx

```diff
@@ -14,6 +14,8 @@ import TabItem from "@theme/TabItem";
 
 API tokens provide authenticated access to HyperSync services, enabling enhanced capabilities and usage tracking. While currently optional, they will become mandatory in the future and offer significant advantages for serious usage.
 
+**Starting from 9 May**, you will still be able to use HyperSync without API tokens, but **rate limits will be applied**. The service will remain **free to use until the end of May**. From **June onwards**, we will introduce a set of **tiered packages based on usage**. Indexers deployed to our hosted service will have special access to HyperSync that does not require a custom API token.
```

Comment:
Should probably update a message in `/hyperindex/messages`

Reply:
Good idea 👍🏼

Reply:
I wonder if we should add some kind of note about API tokens in the HyperIndex docs also.

Reply:
Done 👍🏼